android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/DefaultRepository.kt

```diff
@@ -113,10 +133,37 @@ constructor(open val fhirEngine: FhirEngine, open val dispatcherProvider: Dispat
 
   suspend fun getBinary(id: String): Binary = fhirEngine.get(id)
 
-  suspend fun save(resource: Resource) {
+  suspend fun create(vararg resource: Resource): List<String> {
+    val syncStrategy = mutableMapOf<String, List<String>>()
+
+    sharedPreferencesHelper.read<List<String>>(
+        key = SharedPreferenceKey.PRACTITIONER_DETAILS_CARE_TEAM_IDS.name
```

Comment:
Why do we use different shared preferences keys from the sync strategy key thus enabling the code repetition? If we needed the suffix **_IDS** for the sharedPref keys that could have been done appropriately in code. I also think it does not matter what name we give them as long as they are unique.

Reply:
Okay, I was thinking to use the SyncStrategy.value() as the key.
What do you think?

Reply:
I think the answer for this is dependent on my question whether we would want to have the `SyncStrategy` as part of `ApplicationConfiguration` @dubdabasoduba

Reply:
Quoting from Ben:
> Yes, we can configure them. I think that will give us more flexibility if we needed to change them in future.

So this is confirmed to be configured.